CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* [#12717](https://github.com/cosmos/cosmos-sdk/pull/12717) Use injected encoding params in simapp.
```

Comment:
could you add an entry into upgrading.md file

Reply:
Done :)